.pre-commit-config.yaml

```diff
@@ -1,3 +1,5 @@
+default_install_hook_types: [pre-commit, pre-push]
```

Comment:
> Run pre-commit on pre-push by default

I don't think that's what this does. This just means that all hooks will run pre-push as well, *if you have installed the pre-push hook*, which isn't the default.

Running pre-commit hooks pre-push is also surprising/unusual. I have no objection to people choosing to run them pre-push instead, but that feels like a very personal choice.

Reply:
> This means that all hooks will be installed for pre-push as well as pre-commit.

That is what I meant. 

But no probs - will drop this commit.